apps/middleman/src/lib/dal.ts

```diff
@@ -106,3 +109,15 @@ export async function getBootstrapStatus() {
 
   return { isBootstrapped: false, step: null };
 }
+
+export async function triggerTemporalWorkflow() {
+  const client = getTemporalClient();
+  const config = getTemporalConfig();
+
+  await client.workflow.start(TestWorkflow, {
+    taskQueue: config.taskQueue,
+    workflowId: "test-workflow",
+  });
+
+  return "success";
+}
```

Comment:
:thinking: I don't think this belongs in the "dal" if it stands for data-access-layer. Also, if this code was just for validating temparoal, I'd suggest we remove it.
```suggestion
```

Reply:
Mala mia, se me quedo cuando probaba que el client de temporal funcionaba en el app.